libs/auth/src/mock_file_card.test.ts

```diff
@@ -108,7 +108,7 @@ test('MockFileCard basic mocking', async () => {
         user: electionManagerUser,
       },
     },
-    data: Buffer.from(electionData, 'utf-8'),
+    data: undefined,
```

Comment:
Do we need the `data` prop at all now?

Reply:
Yeah good q! While we don't write election definitions to cards any more, we do still read/write scanner report data:

https://github.com/votingworks/vxsuite/blob/7058c9ab6e5eddaf417d05537a0c35ef249a5363/apps/mark/backend/src/app.ts#L162-L164

So the card interface still needs methods for reading and writing card data, and by extension, the mock card implementation of the card interface still needs to support that, too